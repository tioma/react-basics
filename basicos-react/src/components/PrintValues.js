import React from 'react';

const PrintValues = ({ value, value2 }) => (
  <>
    <h2>{`Este es el primer valor: ${value}`}</h2>
    <h2>{`Este es el segundo valor: ${value2}`}</h2>
  </>
);

export default PrintValues;
