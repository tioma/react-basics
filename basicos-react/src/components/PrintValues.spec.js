import React from 'react';
import { shallow } from 'enzyme';

import PrintValues from './PrintValues';

describe('<PrintValues />', () => {
  it('should match snapshot', () => {
    const component = shallow(<PrintValues />);

    expect(component).toMatchSnapshot();
  });
});
