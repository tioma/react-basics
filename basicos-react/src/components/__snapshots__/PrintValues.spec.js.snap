// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PrintValues /> should match snapshot 1`] = `
<Fragment>
  <h2>
    Este es el primer valor: undefined
  </h2>
  <h2>
    Este es el segundo valor: undefined
  </h2>
  <h3>
    el country es undefined
  </h3>
  <button
    type="button"
  >
    Mostrar mensaje
  </button>
  <OtroComponentcustom />
</Fragment>
`;
